scripts/build-repos.py

```diff
@@ -35,9 +35,11 @@ class PixiBuildError(Exception):
     pass
 
 
-def run_command(cmd: list[str], cwd: Path | None = None) -> tuple[int, str, str]:
+def run_command(
+    cmd: list[str], cwd: Path | None = None, capture_output: bool = False
+) -> tuple[int, str, str]:
     """Run a command and return exit code, stdout, and stderr."""
-    result = subprocess.run(cmd, cwd=cwd, capture_output=True, text=True)
+    result = subprocess.run(cmd, cwd=cwd, capture_output=capture_output, text=True)
```

Comment:
I think that is too noisy as a default.

I'd be happy to introduce a verbose mode though!

Reply:
ok! fair point. changing to True default